docs/semgrep-cloud-platform/dashboard.md

```diff
@@ -25,31 +25,31 @@ Object.entries(frontMatter).filter(
 
 ![Screenshot of dashboard view](/img/dashboard-view.png)
 
-The Findings Dashboard is an overview of your organization’s security posture from data aggregated within Semgrep Cloud Platform. With these metrics you can:
+The Semgrep Cloud Platform Dashboard is an overview of your organization’s security posture from data aggregated within the Semgrep Cloud Platform. With these metrics you can:
 
 * View recurring security issues, consequently taking action on them.
 * Improve communication between developer teams and security teams.
 * Detect vulnerabilities early, thereby preventing these from persisting through to the next stage of product delivery, such as QA.
 
-You can access the Dashboard by [logging into Semgrep Cloud Platform](https://semgrep.dev/login?return_path=/manage/projects).
+You can access the Dashboard by [logging into the Semgrep Cloud Platform](https://semgrep.dev/login?return_path=/manage/projects).
 
 ## Assessing security readiness at a glance
 
-![Screenshot of dashboard header](/img/dashboard-header.png)
+![Screenshot of dashboard Code widget](/img/dashboard-code.png)
 
-The **Header widget** displays high-level security analytics across your entire organization. This includes:
+The **Code** widget displays high-level security analytics across your entire organization. This includes:
```

Comment:
Just FYI, the "Code" and "Supply Chain" titles are titles of the two sides of the page. The header widgets have no title of their own. You can choose to call them Code widget / Supply Chain widget if you want.